File: javascript/discordjs-v13/src/comandos/utilidades/ping.js
//Command ping
//Exemplo: !ping
//Linguagem usada: js
//Author: aquelemesmoojack#4306
//Versão: Discord.JS v13

module.exports.run = async (bot,message,args) => {
    message.reply("Meu ping é de: " + bot.ws.ping + "ms")
}

File: lua/scripts/comandos/Ping.lua
--Ping command
--Exemplo: !ping
--Linguagem usada: lua
--Author: aquelemesmoojack

return {
    name = 'ping',
    aliases = 'ms',
    commandRun = function(bot, msg, args, rest)
        msg.channel:send("Pong!")
    end
}
